xpallets/support/src/serde.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019-2020 ChainX Project Authors. Licensed under GPL-3.0.
+
+use serde::{de, ser, Deserialize};
+
+/// Hex serialization/deserialization
+pub mod hex {
```

Comment:
I suggest renaming `hex` to `serde_hex` directly which allows you not having to use `::hex`, not a fan of `::hex`.

Reply:
Done